src/Payum/Request/Api/SaveCreditCard.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Payum\Request\Api;
+
+use Payum\Core\Request\Generic;
+
+class SaveCreditCard extends Generic
```

Comment:
I see in others Generics we are using getters to retrieve values, may I ask why you didn't?